cloud/spawn.go

```diff
@@ -243,7 +228,25 @@ func CheckInstanceTypeValid(ctx context.Context, d distro.Distro, requestedType
 	return nil
 }
 
-func SetHostRDPPassword(ctx context.Context, env evergreen.Environment, host *host.Host, password string) error {
+// SetHostRDPPassword is a shared utility function to change the password on a windows host
+func SetHostRDPPassword(ctx context.Context, env evergreen.Environment, h *host.Host, pwd string) (int, error) {
```

Comment:
Why is this part of this refactor if it's not in a service or graphql package?

Reply:
All of this validation was previously in graphql but was called UpdateHostPassword. I liked this naming better so I just swapped which one was the public usable function name and which was private.

Reply:
ah nice!